backend/company_api.py

```
@@ -1028,6 +1028,296 @@ async def cancel_onboarding(
+        raw = await call_llm(
+            messages=[
+                {"role": "system", "content": "You return only valid JSON arrays of task objects. No explanation."},
+                {"role": "user", "content": prompt},
+            ],
+            temperature=0.3,
+        )
+
+        import re
+        json_match = re.search(r"\[.*\]", raw.strip(), re.DOTALL)
+        if json_match:
+            suggested_tasks = json.loads(json_match.group())
```

Ruff (0.15.15)

[error] Undefined name `json`

(F821)